packages/core/src/Table/TableCellHead/Sorter.js

```diff
@@ -0,0 +1,64 @@
+import { requiredIf } from '@dhis2/prop-types'
+import { colors } from '@dhis2/ui-constants'
+import PropTypes from 'prop-types'
+import React from 'react'
+import { Action } from './Action'
+
+export const ASC = 'asc'
+export const DESC = 'desc'
+export const DEFAULT = 'default'
+export const SORT_DIRECTIONS = [DEFAULT, ASC, DESC]
+
+export const getNextSortDirection = currentDirection => {
+    const currentIndex = SORT_DIRECTIONS.indexOf(currentDirection)
+    const endIndex = SORT_DIRECTIONS.length - 1
```

Comment:
`endIndex` and the ternary operator can be replaced with modulus:
```suggestion
    const nextIndex = (currentIndex + 1) % SORT_DIRECTIONS.length
```